shell/common/node_bindings.cc

```diff
@@ -136,7 +136,51 @@ bool IsPackagedApp() {
 #endif
 }
 
+// Initialize Node.js cli options to pass to Node.js
+// See https://nodejs.org/api/cli.html#cli_options
+void SetNodeCliFlags() {
+  // Only allow DebugOptions in non-ELECTRON_RUN_AS_NODE mode
+  std::unordered_set<base::StringPiece, base::StringPieceHash> allowed = {
+      "--inspect",          "--inspect-brk",
+      "--inspect-port",     "--debug",
+      "--debug-brk",        "--debug-port",
+      "--inspect-brk-node", "--inspect-publish-uid",
+  };
+
+  // TODO(codebytere): We need to set the first entry in args to the
+  // program name owing to src/node_options-inl.h#L286-L290 but this is
+  // redundant and so should be refactored upstream.
+  const auto argv = base::CommandLine::ForCurrentProcess()->argv();
+  std::vector<std::string> args;
+  args.reserve(argv.size() + 1);
+  args.push_back("electron");
+
+  for (const auto& arg : argv) {
+#if defined(OS_WIN)
+    std::string option = base::UTF16ToUTF8(arg);
+#else
+    std::string option = arg;
+#endif
+    auto stripped = option.substr(0, option.find("="));
+    if (allowed.find(stripped) != allowed.end())
```

Comment:
```suggestion
    if (allowed.count(stripped) != 0)
```

_(nit)_ your version is fine too but this is more idiomatic. (In C++20 we'll have `includes()` which is equivalent to count()!=0)